mm/ktsan/shadow.c

```diff
@@ -68,6 +57,18 @@ void ktsan_free_page(struct page *page, unsigned int order)
 	kt_atomic64_fetch_add_no_ktsan(&kt_shadow_pages,
 		-(1 << (order + KT_SHADOW_SLOTS_LOG)));
 
+	page_addr = (unsigned long)page_address(page);
```

Comment:
I don't understand this. When we add sync objects to memblocks, we don't check that shadow is mapped. So we leak sync objects in pages without shadow or what?